### PR TITLE
add security section to readme

### PR DESCRIPTION
**Pull request by kun2985**
# Description

Add security section to README file

## JIRA

https://jira.canaltp.fr/browse/BOT-360

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,6 +115,18 @@ docker-compose logs -f
 Chaos will be accessible on http://chaos_ws_1.docker if you are using the [docker-gen-hosts tool](https://github.com/vincentlepot/docker-gen-hosts), it will also be accessible on http://chaos-ws.local.canaltp.fr 
 The database will be accessible at 'chaos_database_1.docker' and default RabbitMQ interface at 'http://chaos_rabbitmq_1.docker:15672'.
 
+## Security (optional)
+
+If you want to add more security, you can add a file chaos/clients_tokens.json with the client name and navitia tokens like:
+```
+{
+   "client_name": [
```
Comment (pthegner) [inline README.md]: Could it be the customer id in place of client name ?

Comment (pthegner) [reply README.md]: Or is it a contributor id ?

File: README.md [hunk 2]
```diff
@@ -115,6 +115,18 @@ docker-compose logs -f
 Chaos will be accessible on http://chaos_ws_1.docker if you are using the [docker-gen-hosts tool](https://github.com/vincentlepot/docker-gen-hosts), it will also be accessible on http://chaos-ws.local.canaltp.fr 
 The database will be accessible at 'chaos_database_1.docker' and default RabbitMQ interface at 'http://chaos_rabbitmq_1.docker:15672'.
 
+## Security (optional)
+
+If you want to add more security, you can add a file chaos/clients_tokens.json with the client name and navitia tokens like:
```
Comment (Netmisa) [inline README.md]: I think you can specify in additionally of `(optional)`, If `chaos/clients_tokens.json` file was not found, security is disabled.

**Comment by tchevily** [comment]
replaced with https://github.com/CanalTP/Chaos/pull/266

**Pull request closed without merging**